app/services/email_output_service.rb

```diff
@@ -0,0 +1,70 @@
+class EmailOutputService
+  def initialize(email_service:)
+    @email_service = email_service
+  end
+
+  def execute(submission_id:, action:, attachments:, pdf_attachment:)
+    email_attachments = []
+
+    if action.fetch(:include_attachments) == true
+      email_attachments = email_attachments.concat attachments
+    end
+    if action.fetch(:include_pdf) == true
+      email_attachments << pdf_attachment
+    end
+
+    if email_attachments.empty?
+      send_single_email(
+        action: action,
+        attachments: [],
+        subject: subject(
+          subject: action.fetch(:subject),
+          current_email: 1,
+          number_of_emails: 1,
+          submission_id: submission_id
+        )
+      )
+    else
+      send_emails_with_attachments(action, email_attachments, submission_id: submission_id)
+    end
+  end
+
+  private
+
+  def send_emails_with_attachments(action, email_attachments, submission_id:)
+    email_attachments.each_with_index do |email_attachment, index|
+      send_single_email(
```

Comment:
I would have assumed that send_email is sending a single email

Reply:
thought that methods with `send_email` and `send_emails` in the same class was not readable enough